src/DynamoCoreWpf/Views/PackageManager/Components/PackageManagerWizard/PackageManagerWizard.xaml.cs

```diff
@@ -532,6 +546,19 @@ internal void ClearMarkdownContent()
             }
         }
 
+        internal void CompatibilityMap()
+        {
+            var compatibilityMapList = PackageManagerClient.CompatibilityMapList(); // Fetch full compatibility map
+            var dynamoVersion = VersionUtilities.Parse(DynamoModel.Version).ToString();
+            var host = new
+            {
+                name = DynamoModel.HostAnalyticsInfo.HostProductName,
```

Comment:
@zeusongit For the compatibility matrix, do we use the hostname(which would be something like dynamo revit) or the host product name which will have the host application name(Revit)?

Reply:
For the record, we use the exact same values as when we calculate the compatibility package details.